php/libraries/TimePoint.class.inc

```diff
@@ -194,6 +194,7 @@ class TimePoint
      *                                    being created. If null, consecutive V1, V2,
      *                                    etc will be created.
      * @param \Site       $psc            The center of the timepoint
+     * @param string      $lang           The language code
```

Comment:
`string` -> `?string`

Reply:
are you suggesting that it should be optional?

I think not null is a better idea. we can get the front end to default to a language when only one is available but this part I think should require a lang

Reply:
It was not a suggestion. The function signature has `?string $lang` so the doc comment should be the same.
It could be `string $lang` but the function signature would need to be changed.

Reply:
oh okai.

@jesscall had you considered making lang not optional ?

Reply:
oh oops, I'll change the function signature. I think lang should be required.

Reply:
I think so too, make sure that all calls to this function are including a language

Reply:
You could also make it non-nullable but provide a default value for the argument.